### DISCUSSION TITLE
xuexiluxian/

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#5] [Announcements]
# xuexiluxian/

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/xuexiluxian/

**Comment by rao548** [comment]
学习路线，没有下一步

**Reply by itwanger** [reply]
什么意思呢？

**Comment by CoMList** [comment]
点击每一钟语言的学习路线都是404啊，大佬

**Reply by CoMList** [reply]
这里的学习路线链接好像都有问题，但是页面左上“三”菜单里的路线没问题

**Reply by itwanger** [reply]
已经修复了，原来的路径写错了。

**Comment by zonekoko** [comment]
有没有算法的学习路线啊

**Reply by itwanger** [reply]
收到，马上开一篇出来！

**Reply by zonekoko** [reply]
二哥真顶👍
